src/layouts/TeamLayout/TeamLayout.vue

```diff
@@ -30,6 +30,7 @@ import richie from '@/assets/images/team/Richie.jpg';
 import brittany from '@/assets/images/team/Brittany.jpg';
 import yel from '@/assets/images/team/Gamaliel.jpg';
 import steve from '@/assets/images/team/Steve.jpg';
+import gage from '@/assets/images/team/Gage.jpg';
```

Comment:
built version color is different :(

Reply:
Matched the background of the others and resized so build doesn't overtly update it. Hopefully it works.